format_vectors.py

```diff
@@ -64,7 +68,8 @@ def entry_mode_value(entry):
 aead_idAES128GCM = 0x0001
 aead_idAES256GCM = 0x0002
 aead_idChaCha20Poly1305 = 0x0003
-aeadMap = {aead_idAES128GCM: "AES-128-GCM", aead_idAES256GCM: "AES-256-GCM", aead_idChaCha20Poly1305: "ChaCha20Poly1305"}
+aead_idExportOnly = 0xFFFF
+aeadMap = {aead_idAES128GCM: "AES-128-GCM", aead_idAES256GCM: "AES-256-GCM", aead_idChaCha20Poly1305: "ChaCha20Poly1305", aead_idExportOnly: "Export-Only AEAD"}
```

Comment:
Nit: Please add some line breaks here.  Pretty sure Python allows it.

Reply:
👍